tests/testthat/test-developer-gs_spending_bound.R

```diff
@@ -91,10 +90,17 @@ test_that("Two-sided asymmetric design fails to reproduce gsDesign test.type=3 b
                         upar = list(sf = sfu, total_spend = alpha, param = sfupar),
                         lower = gs_spending_bound,
                         lpar = list(sf = sfl, total_spend = beta, param = sflpar))
-  
+  # compare boundaries
   expect_equal(gsd$upper$bound, (gsdv %>% filter(Bound == "Upper"))$Z, tolerance = 7e-6)
   expect_equal(gsd$lower$bound, (gsdv %>% filter(Bound == "Lower"))$Z, tolerance = 9e-6)
-  expect_equal(gsd$n.I, (gsdv %>% filter(Bound == "Upper"))$info, tolerance = .04) # While tolerance should not be problematic, it seems large
+  
+  # compare statistical information
+  # While tolerance should not be problematic, it seems large
+  expect_equal(gsd$n.I, (gsdv %>% filter(Bound == "Upper"))$info, tolerance = .04) 
+  
+  # compare crossing boundaries probability under null hypothesis (theta = 0)
+  expect_equal((gsdv %>% filter(Bound == "Upper"))$Probability0, sfu(alpha = alpha, t = timing, param = sfupar)$spend)
+  #expect_equal((gsdv %>% filter(Bound == "Lower"))$Probability, sfu(alpha = beta, t = timing, param = sflpar)$spend)
```

Comment:
What's happening with the lines that are commented out? Please either 1) remove them if they are not useful anymore or 2) document the specific reason why they are commented out.

Reply:
It is a pending test. I will revise it later in the test branch.